### PR TITLE
Change the theme to be greyscale.

### LINKED ISSUE #33: Out-GridView on macOS makes it difficult to select objects for the pipeline via -PassThru
Consider the following cmdlet entry:

`Get-AzSubscription | Where-Object -Property State -eq "Enabled" | Out-GridView -PassThru | Select-AzSubscription`

With `Out-GridView`, it's should very easy to graphically select which subscription object from the collection you wish to pass to `Select-AzSubscription`. The code works, but the UI needs to make it clear which row is selected.

For example, it's nearly impossible to tell which row is selected in this image: 
ogvselection
(Sorry, I had to obscure the GUIDs and names.) But the point can still be seen: one of these objects is selected but it's almost impossible to tell.

The colors in the macOS `OGV` UI need adjusting to clearly delineate which object is selected.

### PR DESCRIPTION
**Pull request by corbob**
Fixes #33 

I used the Theme editor from here: https://wieslawsoltes.github.io/ThemeEditor/ to take the light theme and turn it grayscale. I have no idea why the highlight color is still bluish, I could not change it with any of the theme options that were exposed there or in the Avalonia [Light Theme](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Default/Accents/BaseLight.xaml)

Sample output prior to this change:
image

Sample output after this change:
image

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
Would this get rid of the dark theme support?

File: src/OutGridView.Gui/Style.xaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <Thickness x:Key="ThemeBorderThickness">1</Thickness>
+        <sys:Double x:Key="ThemeDisabledOpacity">0.5</sys:Double>
+
+        <sys:Double x:Key="FontSizeSmall">10</sys:Double>
+        <sys:Double x:Key="FontSizeNormal">12</sys:Double>
+        <sys:Double x:Key="FontSizeLarge">16</sys:Double>
+
+        <sys:Double x:Key="ScrollBarThickness">10</sys:Double>
+    </Style.Resources>
+</Style>
```
Comment (TylerLeonhardt) [inline src/OutGridView.Gui/Style.xaml]: nit: newline at end of file.

**Comment by TylerLeonhardt** [comment]
So this file was generated by that tool? That's pretty neat

**Comment by TylerLeonhardt** [comment]
Awesome work!

**Comment by yobyot** [comment]
Thanks.

**Comment by yobyot** [comment]
Did this make it into Microsoft.PowerShell.GraphicalTools for macOS?

Thanks.

**Comment by corbob** [comment]
> Did this make it into Microsoft.PowerShell.GraphicalTools for macOS?
> 
> Thanks.

It doesn't look like there's been a release since this was fixed. Last package on PSGallery is August date, and this was fixed in October.

**Comment by TylerLeonhardt** [comment]
Exactly. Need to do a release soon.

**Comment by yobyot** [comment]
Is there a chance we could get a new build on the Gallery of this to go with pwsh 7 rc1? Windows looks great -- I hate to see the Mac and Linux lose out.

Thanks.

**Pull request merged**